### PR TITLE
More text/cutscene refinements

### PR DESCRIPTION
**Pull request by YourAverageLink**

### CODE REVIEW

File: eventpatches.yaml [hunk 1]
```diff
@@ -1005,6 +1005,11 @@
     cases:
       - Rerandomize Treasure
       - -1
+  - name: Skip Fledge 3
+    type: flowpatch
+    index: 268 # after giving fledge the second stamina potion
```
Comment (CovenEsme) [inline eventpatches.yaml:1010]: Currently, this is only after giving the 1st stamina potion.
Also, cutting out more of the text would be preferable. There's still quite a lot of it to mash through - even with just the one conversation.

File: eventpatches.yaml [hunk 2]
```diff
@@ -1134,6 +1139,11 @@
       type: type1
       next: Kina Quest Switch
       param4: New Kina Text Option
+  - name: Shorten Kina Text
```
Comment (CovenEsme) [inline eventpatches.yaml:1142]: Works well.
Would be great if Eagus' and Pumm's text was also shortened to match.

**Review by CovenEsme — CHANGES_REQUESTED** [review]
Everything works except fledge gives the item after 1 potion instead of 2.
Would be cool if more of the kina and fledge text was skipped but definitely not necessary.

File: eventpatches.yaml [hunk 3]
```diff
@@ -1184,6 +1194,11 @@
     index: 7
     flow:
       next: 21 # never talk about triforce or song of the hero
```
[the remark this answers is not in the document]
Comment (YourAverageLink) [reply eventpatches.yaml:1196]: I think I was trying to skip a long text sequence taht was on 21

**Pull request merged**